src/exampleTypes/Exists.js

```diff
@@ -1,17 +1,34 @@
 import React from 'react'
-import F from 'futil'
+import _ from 'lodash/fp'
 import { contexturify } from '../utils/hoc'
 
-let Exists = ({ tree, node, theme: { RadioList } }) => (
-  <div className="contexture-exists">
-    <RadioList
-      value={node.value ? 'exists' : 'doesNotExist'}
-      onChange={value => {
-        tree.mutate(node.path, { value: value === 'exists' })
-      }}
-      options={F.autoLabelOptions(['exists', 'doesNotExist'])}
-    />
-  </div>
-)
+let getValue = (value, either) =>
+  _.isBoolean(value) ? value : either ? null : false
+
+let Exists = ({
```

Comment:
yeah, you're right - this is line for line identitical with bool, except the className and default display. We should definitely factor this out